### ISSUE TITLE
Empty required vizy field not validated

### ISSUE THREAD

**Issue by dGilli** [verbb/vizy#55]
**Steps to reproduce**
1. Add vizy to field layout and make required.
2. Edit entry and leave field blank. Should be validated with error but saves normally.

**Additional info**
- Plugin version: 1.03
- Craft version: 3.6.17
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.4](https://github.com/verbb/vizy/releases/tag/1.0.4)
